### PR TITLE
Update AWS documentation for accuracy

### PR DESCRIPTION
**Pull request by ystros**
This updates the install and deletion docs for Ops Manager on AWS for accuracy.

* Removes out-of-date screenshots in favor of descriptive text for the current AWS UI.
* Updates screenshots to better match the text (e.g. the word `infrastructure` was used in screenshots, but `management` in the text instructions)
* Updated SSH Load Balancer from a Classic Load Balancer to a Network Load Balancer. (Classic is intended to be deprecated at some point).
* Updated VPC instructions, which now require manually creating an Internet Gateway, a NAT Gateway, and associated Route Tables.
* Update page title so that it isn't used twice for different content ("Installing Ops Manager on AWS")

### CODE REVIEW

**Comment by ystros** [comment]
It appears we don't have permission to push a branch / make a PR against the `docs-book-om` repository. The title change for `aws.html.md.erb` will require you to change this navigation link in the `docs-book-om` repository:
https://github.com/pivotal-cf/docs-book-om/blob/2.10/master_middleman/source/subnavs/om-subnav.erb#L74

```diff
         <span role="button"><strong>Installing on AWS</strong></span>
           <ul>
             <li class="">
-              <a href="https://docs.pivotal.io/ops-manager/<%= product_info['local_product_version'].to_s.sub('.','-') %>/install/aws.html">Installing <%= vars.platform_name %> on AWS</a>
+              <a href="https://docs.pivotal.io/ops-manager/<%= product_info['local_product_version'].to_s.sub('.','-') %>/install/aws.html">Requirements and Prerequisites on AWS</a>
             </li>
             <li class="">
               <a href="https://docs.pivotal.io/ops-manager/<%= product_info['local_product_version'].to_s.sub('.','-') %>/install/aws-iaas-user-roles.html">AWS Permissions Guidelines</a>
```

File: aws/required-objects.html.md.erb [hunk 1]
```diff
@@ -439,10 +377,10 @@ See [Step 5: Configure DNS Records](deploy-manual.html#pcfaws-dns) in _Deploying
 
 You must create a subnet group for RDS named `pcf-rds-subnet-group` from the RDS Dashboard.
 
-See [Step 7: Create RDS Subnet Group](deploy-manual.html#pcfaws-rds-subnet-group) in _Deploying <%= vars.ops_manager %> on AWS_.
+See [Step 6: Create RDS Subnet Group](deploy-manual.html#pcfaws-rds-subnet-group) in _Deploying <%= vars.ops_manager %> on AWS_.
```
Comment (anita-flegg) [inline aws/required-objects.html.md.erb:380]: Do you want the Step number here? You have removed it from nearly every other place.

File: aws/required-objects.html.md.erb [hunk 2]
```diff
@@ -439,10 +377,10 @@ See [Step 5: Configure DNS Records](deploy-manual.html#pcfaws-dns) in _Deploying
...
 
 ## <a id='mysql-db'></a> MySQL Database
 
 You must create a MySQL database from the RDS Dashboard.
 
-See [Step 8: Create a MySQL Database using AWS RDS](deploy-manual.html#pcfaws-mysql-rds) in _Deploying <%= vars.ops_manager %> on AWS_.
+See [Step 7: Create a MySQL Database using AWS RDS](deploy-manual.html#pcfaws-mysql-rds) in _Deploying <%= vars.ops_manager %> on AWS_.
```
[the remark this answers is not in the document]
Comment (ystros) [reply aws/required-objects.html.md.erb:386]: Re: step numbers, the `prepare-env-manual.html` page had its step numbers removed in 2020, but many of the links still had a step number in them.  Contrast that with the `deploy-manual.html` page, which still has the step numbers on the page, so it somewhat makes sense for the step numbers to remain in the links.

We certainly could make this consistent by removing the step numbers both from that page and corresponding links. I don't have a strong opinion on that though.

**Comment by anita-flegg** [comment]
I've made the requested change to the subnav in docs-book-om.

**Pull request merged**